app/src/main/res/layout/activity_achievements.xml

```diff
@@ -0,0 +1,236 @@
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content">
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:orientation="vertical">
+        <include
+            android:id="@+id/toolbar"
+            layout="@layout/toolbar"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content" />
+        <RelativeLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_below="@+id/toolbar"
+            android:background="#D6DCE0"
+            android:orientation="vertical">
+
+            <TextView
+                style="?android:textAppearanceLarge"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginLeft="@dimen/activity_margin_horizontal"
+                android:layout_marginStart="@dimen/activity_margin_horizontal"
+                android:layout_marginTop="@dimen/activity_margin_horizontal"
+                android:text="@string/level"
+                android:id="@+id/achievement_level"/>
+
+            <ImageView
+                android:id="@+id/achievement_badge"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_below="@id/achievement_level"
+                android:layout_centerHorizontal="true"
+                android:layout_marginVertical="@dimen/activity_margin_vertical" />
+
+            <RelativeLayout
+                android:id="@+id/layout_image_uploaded"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_below="@+id/achievement_badge"
+                android:layout_marginEnd="@dimen/activity_margin_horizontal"
+                android:layout_marginLeft="@dimen/activity_margin_horizontal"
+                android:layout_marginRight="@dimen/activity_margin_horizontal"
+                android:layout_marginStart="@dimen/activity_margin_horizontal">
+
+                <TextView
+                    style="?android:textAppearanceMedium"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginLeft="@dimen/activity_margin_horizontal"
+                    android:layout_marginStart="@dimen/activity_margin_horizontal"
+                    android:text="@string/images_uploaded" />
+
+                <com.dinuscxj.progressbar.CircleProgressBar
+                    android:layout_width="35dp"
+                    android:layout_height="35dp"
+                    android:layout_alignParentRight="true"
+                    android:layout_marginRight="32dp"
+                    android:progress="50"
+                    app:progress_end_color="#8C8B98"
```

Comment:
Its a good practice to define these colors in colors.xml file.

Reply:
I will update all these changes ! Sorry for not paying attention to them earlier :)